component.css

```diff
@@ -15,15 +21,24 @@
     --SUCESS-COLOR:#65a30d;
     --INFO-COLOR:#2563eb;
     --WARNING-COLOR:#ca8a04;
+    /* Font-Family */
+    --MAIN-FONT:'Merriweather', serif;
 }
 
 body{
     margin: 0;
     padding: 0;
     box-sizing: border-box;
+    font-family: var(--MAIN-FONT);
 }
 
 .container{
     margin: auto;
     max-width: 900px;
```

Comment:
use rem to stay consistent

Reply:
I used absolute value for the max-width of a container so that the computed width of an element doesn't exceed and stay inside a fix width.